nodejs18.x/hello-gql/{{cookiecutter.project_name}}/greeter/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "hello_world",
+  "version": "1.0.0",
+  "description": "hello world sample for NodeJS",
+  "main": "app.js",
+  "repository": "https://github.com/awslabs/aws-sam-cli/tree/develop/samcli/local/init/templates/cookiecutter-aws-sam-hello-nodejs",
+  "author": "SAM CLI",
+  "license": "MIT",
+  "dependencies": {
+    "axios": ">=0.21.1"
```

Comment:
Do we need axios as a dependency?

Reply:
Good call, I copied from the existing hello-world function and didn't pay attention to what dependencies are used. I'll remove all that are not used